x-pack/plugins/ml/public/application/components/import_export_jobs/export_jobs_flyout/export_jobs_flyout.tsx

```diff
@@ -65,6 +67,10 @@ export const ExportJobsFlyout: FC<Props> = ({ isDisabled, currentTab }) => {
     [toasts]
   );
 
+  const [jobDependencies, setJobDependencies] = useState<JobDependencies>([]);
+  const [selectedJobDependencies, setSelectedJobDependencies] = useState<JobDependencies>([]);
+  // const [showJobDependenciesWarning, setShowJobDependenciesWarning] = useState(false);
```

Comment:
Is this comment needed?

Reply:
Fixed in https://github.com/elastic/kibana/pull/107416/commits/6b533f4b5adf3c972a291bd2c32333c7b2ebb2ea